packages/dx-react-chart/docs/reference/argument-axis.md

```diff
@@ -27,7 +27,8 @@ tickSize? | number | 5 | The tick size.
 position? | 'bottom' &#124; 'top' | 'bottom' | The axis position.
 name? | string | | The axis name.
 indentFromAxis? | number | 10 | The indent from the axis.
-type? | 'band' &#124; 'linear' | | Axis type.
+type? | 'band' &#124; 'linear' | 'linear' | Axis type.
+tickForamt? | (scale: object) => (tick: string) => string  | | A function that returns a tick formatter function.
 rootComponent | ComponentType&lt;[ArgumentAxis.RootProps](#argumentaxisrootprops)&gt; | |  A component that renders the axis root layout.
 tickComponent | ComponentType&lt;[ArgumentAxis.TickProps](#argumentaxistickprops)&gt; | | A component that renders a tick.
 labelComponent | ComponentType&lt;[ArgumentAxis.LabelProps](#argumentaxislabelprops)&gt; | | A component that renders the axis label.
```

Comment:
"that renders the axis label" Should this be 'that renders the **axis's** label'?